src/data/screenshots_de.json

```
@@ -5,16 +5,13 @@
     },
     "section-main": {
         "headline": {
-            "title": "Machen Sie sich selbst ein Bild (Version 2.13)"
+            "title": "Machen Sie sich selbst ein Bild (Version 2.14)"
         },
         "content": {
             "textblock": [
-                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.13 (verfügbar seit 3 November 2021) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
+                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.14 (verfügbar seit 22 November 2021) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
                 "Mit diesem Update stellen wir neue und erweiterte Funktionen zur Verfügung:",
-                "<b>Lokale 7-Tage-Inzidenz Hospitalisierung:</b> Auf den lokalen Kacheln zur 7-Tage-Inzidenz der Neuinfektionen wird nun zusätzlich die 7-Tage-Inzidenz der Hospitalisierung angezeigt. Da diese Kennzahl nur auf Bundeslandebene verfügbar ist, wird bei Auswahl eines Kreises die Hospitalisierung zum jeweiligen Bundesland angezeigt.",
-                "<b>Änderung des Links zu Häufige Fragen:</b> Wenn Sie in der App die häufigen Fragen aufrufen, werden Sie nun auf Seiten unter coronawarn.app geleitet. Dort finden Sie mehr Details zu Ihren Fragen.",
-                "<b>Zuordnung von Zertifikaten:</b> Die Zuordnung von Zertifikaten zu Personen wurde fehlertoleranter gestaltet. Die App ignoriert nun eine unterschiedliche Anzahl von Leerzeichen im Namen, sodass die Zertifikate unabhängig davon der gleichen Person zugeordnet werden können.",
-                "<b>Zertifikate wiederherstellen:</b> Entfernte Zertifikate werden in einen Papierkorb verschoben. Sie können sie innerhalb von 30 Tagen wiederherstellen.",
+                "<b>Papierkorbfunktion für Tests:</b> Sie können in der App Schnelltests und PCR-Tests nach dem Löschen wiederherstellen. Sie haben somit die Möglichkeit Objekte die sie versehentlich gelöscht haben, wiederherzustellen.",
```

```suggestion
                "<b>Papierkorbfunktion für Tests:</b> Sie können in der App Schnelltests und PCR-Tests nach dem Löschen wiederherstellen. Sie haben somit die Möglichkeit, Objekte, die Sie versehentlich gelöscht haben, wiederherzustellen.",
```

Check with https://mentor.duden.de/.